src/network/clientpackethandler.cpp

```diff
@@ -1330,6 +1330,10 @@ void Client::handleCommand_SrpBytesSandB(NetworkPacket* pkt)
 void Client::handleCommand_CSMFlavourLimits(NetworkPacket *pkt)
 {
 	*pkt >> m_csm_flavour_limits >> m_csm_noderange_limit;
+
+	// Now we have flavours, load mods if it's enabled
+	// Note: this should be moved after mods receptions from server instead
+	loadMods();
```

Comment:
It might be good to leave a note in the documentation that if the server doesn't send flavour limits, then no client mods will load.

Reply:
@Ekdohibs exact, it's also a interesting side effect, but i should move that outside of flavour, after pushing mods from server packet (when feature was added

Reply:
With the current implementation the server can also force client-side mods to be loaded, read twice.
The client should probably only accept the `CSMFlavourLimits` packet once.

Reply:
@sfan5 exact you are right i will permit to accept packet twice but not loading mods, it will permit to adjust flavours dynamically at runtime if needed later (we can add a SSM interface to manipulate the flag and change flavours)

Reply:
> Note: this should be moved after mods receptions from server instead

This sounds like an easy task for a beginning contributor, is it? Should I try? What were the obstacles that it's still a todo?